tendermint/spn
# fix(monitoring/simulation): fix simulation for the monitoring modules

### What does this PR does?

- Add default params to the monitoring provider simulation;
- Add default params to the monitoring consumer simulation;
- Add default `PortID` to the monitoring provider genesis to run the simulation;
- Add default `PortID` to the monitoring consumer genesis to run the simulation;
- Fix some lint issues;

### How to test?

- Create a chain from this branch https://github.com/tendermint/starport/pull/2157 and run the simulation:
```shell
starport c simulate --numBlocks 200 --blockSize 50 --seed 3
```

- The simulation should fail because the parameters are missing. import this SPN version and try again:
```shell
go get -u github.com/tendermint/spn@0ec63f7 && go mod tidy && \
starport c simulate --numBlocks 200 --blockSize 50 --seed 3
```
